build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         classpath 'com.google.gms:google-services:2.1.0'
     }
 }
+project.ext.fabricKey = "";
```

Comment:
Basically the build freaks out if there's no Fabric key. Not sure if you get this but it throws an error on a gradle sync stating that `fabricKey` doesn't exist